pkgs/applications/misc/get_iplayer/default.nix

```diff
@@ -22,11 +22,14 @@ perlPackages.buildPerlPackage rec {
   outputs = [ "out" "man" ];
 
   installPhase = ''
+    runHook preInstall
     mkdir -p $out/bin $out/share/man/man1
     cp get_iplayer $out/bin
     wrapProgram $out/bin/get_iplayer --suffix PATH : ${lib.makeBinPath [ atomicparsley ffmpeg flvstreamer rtmpdump ]} --prefix PERL5LIB : $PERL5LIB
     cp get_iplayer.1 $out/share/man/man1
+    runHook postInstall
   '';
+
```

Comment:
This could be a seperate commit with subject "get_iplayer: Fix: Add missing runHook calls"